Compute only unique minterms during mintermization

jurajsic opened the pull request:
This PR should improve the speed of mintermization, by computing minterms only for unique BDDs from transitions + only keeping unique minterms.

jurajsic committed:
reduce number of minterms

tfiedor commented on src/mintermization.cc:
```diff
@@ -115,37 +115,30 @@ void Mata::Mintermization::trans_to_bdd_afa(const IntermediateAut &aut)
-    std::vector<BDD> stack;
-    // we start mintermization with first bdd and its negation
-    stack.push_back(bdds.front());
-    stack.push_back(!bdds.front());
```
What about the mentioned negation? Shouldn't it be added as well? Or is it included in the `bdd_mng.bddOne()`?

jurajsic replied:
BDD at front is now processed in the following loop where we do both `1 && front` and `1 && !front`, so we add it to the set of minterms

tfiedor reviewed (approved):
Seems OK

Adda0 reviewed (approved):
The changes look good to me. We will fix the Python binding once the PR is ready to be merged.

jurajsic commented:
Is there mintermization in python binding? I changed the interface a bit, so it could break it there. I also changed one header file in cudd, I am not sure whether it should be also changed in `cudd-min` for binding.

tfiedor commented:
> Is there mintermization in python binding? I changed the interface a bit, so it could break it there. I also changed one header file in cudd, I am not sure whether it should be also changed in `cudd-min` for binding.

Should I fix the binding? You have nothing more to commit?

tfiedor committed:
binding: (hopefully) fixed the binding

tfiedor commented:
The binding should be fixxed.

Adda0 commented:
I am fine with the changes. Do any of you plan to add anything new to the PR, or can we merge the PR?

tfiedor commented:
I won't be adding any new changes to this PR,
